source/plugin/text/text.rst

```diff
@@ -3,136 +3,125 @@ Creating Text
-in text by appending additional texts with different colors:
+Any Minecraft color specified within the ``NamedTextColor`` class can be used when coloring text, as well as full-RGB colors using ``TextColor``.
+Multiple colors can be used in text by appending additional texts with different colors:
 
 Example: Multi-colored Text
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. code-block:: java
 
-    Text multiColoredText = Text.builder("Sponges are ").color(TextColors.YELLOW).append(
-            Text.builder("invincible!").color(TextColors.RED).build()).build();
+    import net.kyori.adventure.text.format.TextColor;
+
+    Component multiColoredText = Component.text("Sponges are ").color(NamedTextColor.YELLOW).append(
+            Component.text("invincible!").color(TextColor.color(0x5fb0ff)));
 
 Styling
-~~~~~~~
+=======
 
-The builder can also be used to style text, including underlining, italicizing, etc.
+The API can also be used to style text, including underlining, italicizing, etc. This is called "decorating" in the Adventure library.
 
 Example: Styled Text
 ~~~~~~~~~~~~~~~~~~~~
 
 .. code-block:: java
 
-    import org.spongepowered.api.text.format.TextStyles;
+    import net.kyori.adventure.text.format.TextDecoration;
 
-    Text styledText = Text.builder("Yay! Styled text!").style(TextStyles.ITALIC).build();
+    Component styledText = Component.text("Yay! Styled text!").decorate(TextDecoration.ITALIC);
 
 Just like with colors, multiple styles can be used by chaining together separately styled texts.
 
 Example: Multi-styled Text
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. code-block:: java
-
-    Text multiStyledText = Text.builder("I'm italicized! ").style(TextStyles.ITALIC)
-            .append(Text.builder("I'm bold!").style(TextStyles.BOLD).build()).build();
-
-Coloring & Styling Shortcut
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The :javadoc:`Text#of(Object...)` method provides a simple way to add color and styling to your text in a much more
-concise way.
-
-Example: Color & Style Shortcut
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. code-block:: java
 
-    Text colorAndStyleText = Text.of(TextColors.RED, TextStyles.ITALIC, "Shortcuts for the win!");
+    Component multiStyledText = Component.text("I'm italicized! ").decorate(TextDecoration.ITALIC)
+            .append(Component.text("I'm bold!").decorate(TextDecoration.BOLD));
 
 .. _text-actions:
 
-Text Actions
-~~~~~~~~~~~~
+Text Events
+===========
 
-The text builder also offers the ability to create actions for text. Any action specified within the
-:javadoc:`TextActions` class can be used when creating text actions for text. The method below is a small example of
-what text actions can do.
+Components also offer the ability to create actions for text. Any action specified within the
+``HoverEvent`` or ``ClickEvent`` classes can be used when creating text actions for text.
+The method below is a small example of what text actions can do.
```

Comment:
Maybe mention `SpongeComponents.executeCallback` here as an extra click event provided by Sponge?

Reply:
👌. I added a reference to SpongeComponents and and example using the callback component.